weather_dl/download_pipeline/config.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import typing as t
+import dataclasses
+
+Values = t.Union[t.List['Values'], t.Dict[str, 'Values'], bool, int, float, str]  # pytype: disable=not-supported-yet
+
+
+@dataclasses.dataclass
+class Config:
```

Comment:
I see the `Config` dataclass to be the same concern as the rest of the parsers. The parser.py file is meant to parse a config; for that reason, I think the `Config` dataclass should live in that file.

Reply:
To avoid the problem of circular import, we will keep it in a separate file only.